tests/perf/benchmark.py

```diff
@@ -0,0 +1,200 @@
+            result = pd.read_csv(csv_file_path)
+            # Append metadata if any
+            cfg_file_path = Path(csv_file_path).parent / "cfg.yaml"
+            if cfg_file_path.exists():
+                with cfg_file_path.open("r") as cfg_file:
+                    tags = yaml.safe_load(cfg_file).get("tags", {})
+                    for k, v in tags.items():
+                        result[k] = v
+            results.append(result)
+        if len(results) > 0:
+            return pd.concat(results, ignore_index=True)
+        else:
+            return None
+
+    def _build_config(
+        self,
+        model_id: str,
+        train_params: dict = {},
+        tags: dict = {},
+    ) -> dict:
+        """Build config for tools/expeirment.py."""
+        all_train_params = self.train_params.copy()
+        all_train_params.update(train_params)
+        all_tags = self.tags.copy()
+        all_tags.update(tags)
+
+        cfg = {}
+        cfg["tags"] = all_tags  # metadata
+        cfg["output_path"] = os.path.abspath(Path(self.output_root) / "-".join(list(all_tags.values()) + [model_id]))
+        cfg["constants"] = {
+            "dataroot": os.path.abspath(self.data_root),
+        }
+        cfg["variables"] = {
+            "model": [model_id],
+            "data": self.datasets,
+        }
+        cfg["repeat"] = self.num_repeat
+        cfg["command"] = []
+        resource_param = ""
+        if self.track_resources:
+            resource_param = "--track-resource-usage all"
+        if self.num_epoch > 0:
+            self._set_num_epoch(model_id, all_train_params, self.num_epoch)
+        params_str = " ".join([f"--{k} {v}" for k, v in all_train_params.items()])
+        cfg["command"].append(
+            "otx train ${model}"
+            " --train-data-roots ${dataroot}/${data}" + f"/{self.subset_dir_names['train']}"
+            " --val-data-roots ${dataroot}/${data}" + f"/{self.subset_dir_names['val']}"
+            " --deterministic"
+            f" {resource_param}"
+            f" params {params_str}"
+        )
+        cfg["command"].append("otx eval --test-data-roots ${dataroot}/${data}" + f"/{self.subset_dir_names['test']}")
+        if self.eval_upto == "train":
+            return cfg
+
+        cfg["command"].append("otx export")
+        cfg["command"].append("otx eval --test-data-roots ${dataroot}/${data}" + f"/{self.subset_dir_names['test']}")
+        if self.eval_upto == "export":
+            return cfg
+
+        cfg["command"].append("otx optimize")
+        cfg["command"].append("otx eval --test-data-roots ${dataroot}/${data}" + f"/{self.subset_dir_names['test']}")
+        return cfg
+
+    @staticmethod
+    def _set_num_epoch(model_id: str, train_params: dict, num_epoch: int):
+        """Set model specific num_epoch parameter."""
+        if "padim" in model_id:
+            return  # No configurable parameter for num_epoch
+        elif "stfpm" in model_id:
```

Comment:
```suggestion
        elif "stfpm" in model_id or "draem" in model_id:
```
Doesn't `draem` have to be considered?

Reply:
`draem` is experimental template, and it has num_epoch parameter.